### PR TITLE
Homework for lection 6

### PR DESCRIPTION
**Pull request by andriikalinkin**
Updates:
- Employee pagination and email search in EmployeeListView
- Create EmployeeDetailView and employee_detail.html

### CODE REVIEW

File: course_django/asgi.py [hunk 1]
```diff
@@ -11,6 +11,6 @@
 
 from django.core.asgi import get_asgi_application
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'course_django.settings')
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "course_django.settings")
```
Comment (marcustas) [inline course_django/asgi.py:14]: why did you change single quotes to single quotes?

Comment (andriikalinkin) [reply course_django/asgi.py:14]: The replacement occurred not from single to single, but from single to double quotes. In any case, this happened because I used the ` ruff .` command (last commit).
